migration/migrations/course/20180611120540_regrade_request.py

```diff
@@ -1,5 +1,6 @@
 def up(config, conn, semester, course):
     with conn.cursor() as cursor:
+
```

Comment:
Remove whitespace change. Migration files should almost never be modified after their initial creation